src/controllers/dlgprefcontroller.cpp

```diff
@@ -57,7 +57,7 @@ DlgPrefController::DlgPrefController(
     initTableView(m_ui.m_pOutputMappingTableView);
 
     std::shared_ptr<LegacyControllerMapping> pMapping = m_pController->cloneMapping();
-    slotShowMapping(pMapping);
+    slotShowMapping(std::move(pMapping));
```

Comment:
You may consider to use m_pMapping = std::move(pMapping); in the very beginning of slotShowMapping() and use this afterwards.